fetch/fetch.go

```diff
@@ -146,6 +170,8 @@ func DefaultConfig() Config {
 			// ballots > 300 bytes
 			// often queried after receiving gossip message
 			hashProtocol: {Queue: 2000, Requests: 200, Interval: time.Second},
+			// active sets (can get quite large)
+			activeSetProtocol: {Queue: 1, Requests: 1, Interval: 30 * time.Second},
```

Comment:
if someone generates custom activeset they should be able to serve it atleast to several peers on the public network.
maybe it should have atleast the same parameters as ax/1, otherwise this is just kills any chances of delivering such activeset